feature-libs/asm/core/facade/asm.service.ts

```diff
@@ -20,7 +22,24 @@ import { AsmSelectors } from '../store/index';
   providedIn: 'root',
 })
 export class AsmService {
-  constructor(protected store: Store<StateWithAsm>) {}
+  // TODO(#206): make asmFacade required
+  constructor(
+    store: Store<StateWithAsm>,
+    // eslint-disable-next-line @typescript-eslint/unified-signatures
+    asmFacade: AsmCustomerListFacade
+  );
+  /**
+   * @deprecated since 5.1
+   */
+  constructor(store: Store<StateWithAsm>);
+  constructor(
+    protected store: Store<StateWithAsm>,
+    @Optional() protected asmFacade?: AsmCustomerListFacade
+  ) {}
+
+  getCustomerLists(): Observable<QueryState<CustomerListsPage>> | undefined {
```

Comment:
since we have `AsmCustomerListFacade`, so no need add this function in `AsmService`, right? Customer can get this function from `AsmCustomerListFacade`.